dtoolkit/transformer/sklearn/OneHotEncoder.py

```diff
@@ -106,6 +106,9 @@ def __init__(
         )
         self.categories_with_parent = categories_with_parent
 
+        if not SKLEARN_GE_12:
+            self.sparse_output = sparse_output
```

Comment:
There don't have `spare_output` in the lower sklearn version.
So there need to manually add into class attributes.